### PR TITLE
[release/v2.20] Bump version to 2.20.13

### PR DESCRIPTION
**Pull request by pkprzekwas**
**What this PR does / why we need it**:
What the title says.

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**
/kind chore

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

File: src/assets/config/changelog.json [hunk 1]
```diff
@@ -1,21 +1,9 @@
 {
   "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.20.md#v22012",
```
Comment (embik) [inline src/assets/config/changelog.json:2]: Link needs to be updated.

File: src/assets/config/changelog.json [hunk 2]
```diff
@@ -1,21 +1,9 @@
...
-    },
-    {
-        "category": "fixed",
-        "description": "Fixed an issue where creating Clusters through ClusterTemplates was failing without leaving any trace."
-    },
     {
         "category": "fixed",
-        "description": "Fixed an API error in extended disk configuration for the Anexia provider."
+        "description": "Update machine-controller to version v1.54.1 to fix container runtime setup for new nodes"
     }
```
Comment (embik) [inline src/assets/config/changelog.json:7]: Can you please also add an entry for https://github.com/kubermatic/kubermatic/pull/11750? That's a user-facing change in 2.20.13.

**Review by embik — CHANGES_REQUESTED** [review]

File: src/assets/config/changelog.json (hunk 2 above)
Comment (pkprzekwas) [reply src/assets/config/changelog.json:7]: Good point. I run gchl only for dashboard, forgot about kubermatic repo.

**Review by embik — APPROVED** [review]

**Pull request merged**